.rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+reorder_imports = true
+max_width = 100
```

Comment:
It's nice to check this into a Rust project root because people often have their own rustfmt configuration files in their home directories, and over time a lot of pull requests will then get infected with irrelevant refactors that cut long argument lines into several short ones etc... Having this in the root will override the ones that people set in their home directories sometimes, preventing their personal preferences from impacting this project's.